noodles-editor/src/noodles/operators.ts

```diff
@@ -1829,7 +1829,7 @@ export class NetworkOp extends Operator<NetworkOp> {
 export class SwitchOp extends Operator<SwitchOp> {
   static displayName = 'Switch'
   static description =
-    'Switch between multiple values based on an index. Blend enables interpolation between values.'
+    'Select one value from a list using an index (0, 1, 2...). With blend enabled, smoothly interpolate between values for animation effects.'
```

Comment:
The ListField concept is novel and I'm not satisfied yet with how to teach it someone new. It's a list, but of _ops_ not _data_ like you'd typically think.